### DISCUSSION TITLE
Payment Processor API

### DISCUSSION THREAD

**Discussion by olivermrbl** [medusajs/medusa#2849] [Feature Requests]
### What
In our efforts to support a broader range of payment providers, we will introduce a Payment Processor API defining the interface for payment providers with a traditional authorization -> capture flow e.g. Stripe.

**Comment by Horib** [comment]
Do we have an estimated time of delivery on this?

**Reply by olivermrbl** [reply]
We are in the final review stages. You can follow along [here](https://github.com/medusajs/medusa/pull/2978). The following steps will be to migrate all existing payment plugins to use the new API - work has already started in [this PR](https://github.com/medusajs/medusa/pull/3257).
